scripts/check_organizations.py

```diff
@@ -0,0 +1,43 @@
+import pathlib
+import sys
+
+from frictionless import Resource, validate
+from lib.organization import (
+    get_organizations_path,
+    contains_one_organization_per_file,
+    get_organization_validation_report,
+)
+
+import argparse
+from pathlib import Path
+
+
+def main(directory: Path) -> int:
+    code = 0
+    for org_path in get_organizations_path(directory):
+        if not contains_one_organization_per_file(org_path):
+            print(
+                f"an organization file must contain only one organization. Multiple organizations found in {org_path}"
+            )
+            code = 1
+            break
+
+        report = get_organization_validation_report(org_path)
+
+        if report.valid == False:
```

Comment:
(Nit)

```suggestion
        if not report.valid:
```

D'une manière générale en Python on écrit `if <condition>` ou `if not <condition>`, où `<condition>` peut être n'importe quelle expression dont Python évalue la valeur "truth-y".

Ainsi :

```python
items = []

# Equivalent à : if items.__bool__() is False
# Pour les listes, sets, tuples, etc, leur `.__bool__()` renvoie True
# si la collection est non-vide
if not items:
    print("items est vide")

items.append(1)

if items:  # Equivalent à : if items.__bool__() is True
    print("Il y a des éléments")
```

Affichera `items est vide`